### ISSUE TITLE
[Callout] There must be a way to provide role and description (aria-describedby) for the Callouts

### ISSUE THREAD

**Issue by akashperfect** [OfficeDev/office-ui-fabric-react#1038]
### Bug Report

- __Package version(s)__: 1.9.1
- __Browser and OS versions__: All

#### Describe the issue:

There should be a way to provide the role and description (aria-describedby) for the callouts. Either while using them as Non-Modal dialogue or Popup Help in which case role will be specified as dialog.
https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/#dialog_nonmodal
https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/#popuphelp

#### Actual behavior:

No way to provide role and description (aria-describedby) for the callouts

#### Expected behavior:

There should be a way to provide the role and description (aria-describedby) for the callouts.

**Comment by battletoilet** [comment]
This issue has been resolved.
